zbus_macros/tests/tests.rs

```diff
@@ -220,13 +220,17 @@ fn test_interface() {
     <arg name="arg" type="y"/>
     <arg name="other" type="s"/>
   </signal>
-  <property name="MyCustomProperty" type="u" access="readwrite"/>
+  <property name="MyCustomProperty" type="u" access="readwrite">
+    <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="true"/>
```

Comment:
I think `true` is the default? If so, we should skip the annotation when it's `true`. I think other implementations also skip it as otherwise, it will mean a lot of noise in many introspection files.

Reply:
Did you remove the test all together? :thinking: